DisplayOptionsPanel.c

```diff
@@ -90,7 +122,7 @@ const PanelClass DisplayOptionsPanel_class = {
 DisplayOptionsPanel* DisplayOptionsPanel_new(Settings* settings, ScreenManager* scr) {
    DisplayOptionsPanel* this = AllocThis(DisplayOptionsPanel);
    Panel* super = (Panel*) this;
-   FunctionBar* fuBar = FunctionBar_new(DisplayOptionsFunctions, NULL, NULL);
+   FunctionBar* fuBar = FunctionBar_new(BooleanDisplayOptionsFunctions, BooleanDisplayOptionsKeys, BooleanDisplayOptionsEvents);
```

Comment:
What about creating the Panel without a Function Bar at first and just set it based on the default-selected item just before returning? Avoids having this statement go out of sync when the first item changes.

Also maybe rename `Boolean*` to `Checkbox*`?

Reply:
ok

Reply:
Checkbox seems more reasonable @BenBE

Reply:
Will update that